<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="startLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Cantarell&quot;;</string>
   </property>
   <property name="text">
    <string>Start:</string>
   </property>
  </widget>
  <widget class="QLabel" name="endLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Cantarell&quot;;</string>
   </property>
   <property name="text">
    <string>End:</string>
   </property>
  </widget>
  <widget class="QLabel" name="startminLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>66</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Min:</string>
   </property>
  </widget>
  <widget class="QLabel" name="startsecLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>66</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Sec:</string>
   </property>
  </widget>
  <widget class="QLabel" name="endsecLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>380</y>
     <width>66</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Sec:</string>
   </property>
  </widget>
  <widget class="QLabel" name="endminLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>66</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Min:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="startminLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>113</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="startsecLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>113</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="endminLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>113</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="endsecLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>113</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>470</y>
     <width>103</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>103</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Cantarell&quot;;</string>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="startmsLabel">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>240</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Millisec:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="startmsLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>113</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="endmsLabel">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>420</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Millisec:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="endmsLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>410</y>
     <width>113</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
